modules/reloadBidAdapter.js

```diff
@@ -0,0 +1,513 @@
+import {
+  BANNER
+}
+from 'src/mediaTypes';
+import {
+  registerBidder
+}
+from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'reload';
+
+const VERSION_ADAPTER = "1.0";
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: [],
+  supportedMediaTypes: [BANNER],
```

Comment:
Banner is default format. Please remove `supportedMediaTypes` property  if you are only supporting banner

Reply:
Fixed